src/Models/ShallowWaterModels/shallow_water_model.jl

```diff
@@ -35,11 +35,14 @@ function ShallowWaterSolutionFields(grid, bcs, prognostic_names)
     return NamedTuple{prognostic_names[1:3]}((u, v, h))
 end
 
-mutable struct ShallowWaterModel{G, A<:AbstractArchitecture, T, GR, V, U, R, F, E, B, Q, C, K, TS, FR} <: AbstractModel{TS}
+mutable struct ShallowWaterModel{G, A<:AbstractArchitecture, VM, T, GR, RG, N, V, U, R, F, E, B, Q, C, K, TS, FR} <: AbstractModel{TS}
                           grid :: G         # Grid of physical points on which `Model` is solved
+          vertical_formulation :: VM        # Either single-layer or multi-layer
```

Comment:
If we are using `vertical_formulation`, should we use `VF` instead of `VM`, to make it more intuitive?

Reply:
Agreed! I'll change it